<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_conteiner</class>
 <widget class="QWidget" name="Register_conteiner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>40</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1709</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="2,3" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="7" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignLeft">
     <widget class="QCheckBox" name="checkBoxBit7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QLabel" name="labelInterpritation">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit0">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLabel" name="label_hex">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>0x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="labelBit3">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelBit7">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="labelBit2">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="labelBit0">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>F</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="labelBit4">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>F</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelBit6">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="labelBit1">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="1" column="13">
     <widget class="QLabel" name="labelDiscription">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QLabel" name="labelValue">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelBit5">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
      <property name="lable_register_brouser_num_bit" stdset="0">
       <string>true</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="checkBoxBit3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>In</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelRegNum">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Num</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="lable_register_brouser" stdset="0">
       <string>set</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
